cmd/tailscaled/tailscaled.go

```diff
@@ -276,29 +276,13 @@ func main() {
 	}
 
 	if buildfeatures.HasTPM {
-		if !args.encryptState.set {
-			args.encryptState.v = defaultEncryptState()
+		encrypt, err := defaultEncryptState(args.statepath)
```

Comment:
I don't like how this looks like it's looking up a default value even if it's already explicitly set.

Previously it only did this if not set which was nicer, even if the whole thing was kinda long and would be nicer pulled out of `main`